src/js/components/Fragments/Navbuttons.sass

```diff
@@ -0,0 +1,26 @@
+.Navbuttons
+    position: absolute
+    bottom: 20px
+    width: 100%
+    overflow: hidden
+    max-height: 60px
+    background-color: #eee0
+    text-align:center
+.btnNavButton
+    background-color: #333
+    color: white
+    height: 60px
+.leftNavButton
+    border-top-right-radius: 40px
+    border-bottom-right-radius: 40px
+    float: left
+    width: 30%
+.rightNavButton
+    border-top-left-radius: 40px
+    border-bottom-left-radius: 40px
+    float: right
+    width: 30%
+.centerNavButton
+    display: inline-block
+    border-radius: 100%
+    width: 60px
```

Comment:
The proper notation for the css in sass should be the same as normal css for example: 
`.centerNavButton {
    display: inline-block;
    border-radius: 100%;
    width: 60px;
}`
Semi colons and curly backets are important, There should also be a line space in between each class

Reply:
upon reviewing the sass documentation on this url "https://sass-lang.com/guide"
It states that sass has no brackets nor semi colon as it is tab indented like Python code could would be.

Reply:
I see, thank you for the link, as long as it complies I'm happy then :)